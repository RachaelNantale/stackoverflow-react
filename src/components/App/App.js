import React from 'react';
import { BrowserRouter, Route, Switch } from 'react-router-dom';
import Home from '../Home/home';
import AuthPageConnected from '../../containers/auth';
import QuestionPageConnected from '../../containers/postQuestion';
import ViewallQtnsConnected from '../../containers/getQuestions';

const App = () => (
  <BrowserRouter>
    <React.Fragment>
      <Switch>
        <Route exact path="/" component={Home} />
        <Route exact path="/questions" component={ViewallQtnsConnected} />
        <Route exact path="/post_question" component={QuestionPageConnected} />
        <Route path="/signup" component={AuthPageConnected} />

      </Switch>
    </React.Fragment>
  </BrowserRouter>


);


export default App;
