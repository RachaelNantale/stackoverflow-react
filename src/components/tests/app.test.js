import React from 'react';
import { shallow } from 'enzyme';
import App from '../App/App';

it('renders the signup component', () => {
  const AppComponent = shallow(<App />);
  expect(AppComponent).toMatchSnapshot();
});
