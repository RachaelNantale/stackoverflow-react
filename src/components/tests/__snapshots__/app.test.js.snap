// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the signup component 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/questions"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/post_question"
    />
    <Route
      component={[Function]}
      path="/signup"
    />
  </Switch>
</BrowserRouter>
`;
